samples/README.md

```diff
@@ -78,23 +100,30 @@ Note that building all the samples at once is currently only available in the V2
 
 * `-DCMAKE_BUILD_TYPE` and `--config` needs to match the `CMAKE_BUILD_TYPE` when aws-iot-device-sdk-cpp-v2 built. `--config` is only REQUIRED for multi-configuration build tools.
 
-### Sample help
+## Sample help
 
-All samples will show their options by passing in `--help`. For example:
+All samples will show their options and arguments by passing in `--help`. For example:
 
 ```sh
-./build/mqtt5/mqtt5_pubsub/mqtt5_pubsub --help
+./build/mqtt/mqtt5_x509/mqtt5_x509 --help
 ```
-
-Which will result in output showing all of the options that can be passed in at the command line, along with descriptions of what each does and whether or not they are optional or not.
-
-### Enable logging in samples
-
-To enable logging in the samples, you can pass in `--verbosity`, and optionally `--log_file`, to the sample:
-
-```sh
-./build/basic-pub-sub --verbosity "Trace" --log_file "log.txt"
+will result in the following print output:
 ```
+MQTT5 X509 Sample (mTLS)
+options:
+  --help        show this help message and exit
+required arguments:
+  --endpoint    IoT endpoint hostname
+  --cert        Path to the certificate file to use during mTLS connection establishment
+  --key         Path to the private key file to use during mTLS connection establishment
+optional arguments:
+  --client_id   Client ID (default: mqtt5-sample-<uuid>)
+  --topic       Topic (default: test/topic)
+  --message     Message payload (default: Hello from mqtt5 sample)
+  --count       Messages to publish (0 = infinite) (default: 5)
+```
+The sample will not run without the required arguments and will notify you of missing arguments.
+
+## Enable logging in samples
 
-* `--verbosity`: The level of logging shown. Can be `Trace`, `Debug`, `Info`, `Warn`, `Error`, `Fatal` or `None`. Logging will not occur if this is not passed in with `None` or greater logging level.
-* `--log_file`: The filepath to store the logs at. This is optional, and if undefined the logs will be printed to `stdout` instead.
+Instructions to enable logging are available in the [FAQ](../documents/FAQ.md) under [How do I enable logging](../documents/FAQ.md#how-do-i-enable-logging).
```

Comment:
Can we expand the logging section of the FAQ? It's currently bare-bones. A customer should be able to easily add logging to a sample following provided instructions. I think the required `#include` is missing as well as the available logging levels.